Client/Frontend/Widgets/TabsButton.swift

```diff
@@ -43,6 +49,21 @@ struct TabsButtonUX {
 }
 
 class TabsButton: UIControl {
+    var theme: Theme = TabsButtonUX.Themes[Theme.NormalMode]!
+    var highlightButton: Bool = false {
```

Comment:
Nit: Bools should describe a state (`higlighted` or `selected`) rather than an action (`highlightButton`).